model/src/main/java/org/cbioportal/genome_nexus/model/Ann.java

```diff
@@ -0,0 +1,115 @@
+package org.cbioportal.genome_nexus.model;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+public class Ann
+{
+    private String distance_to_feature;
+    private String effect;
+    private String feature_id;
+    private String feature_type;
+    private String gene_id;
+    private String gene_name;
+    private String hgvs_c;
+    private String putative_impact;
+    private String transcript_biotype;
```

Comment:
for other models we use CamelCasing, so prolly good to be consistent

Reply:
hmm actually maybe better to keep it this way and have the returned data have the same field names as the original source. What do you think @onursumer ?

Reply:
@inodb @emz1003 I think we should always stick to the camel case, We can always map field names to whatever we want by using mixins and mongodb annotations. That's actually what we've already done for VEP annotation model.